### PR TITLE
refactor(BranchPrediction): multiple definitions of (un)likely

### LINKED ISSUE #599: The `likely`/`unlikely` annotations are defined in five different source files
We should probably only define it once in a header, rather than in each individual file.

../lib/testing/pulsarSimProcess.cpp
../lib/stages/pulsarPostProcess.cpp
../lib/stages/frbPostProcess.cpp
../lib/stages/beamformingPostProcess.cpp
../lib/hsa/hsaPulsarUpdatePhase.cpp

### PR DESCRIPTION
**Pull request by nritsche**
Closes #599

### CODE REVIEW

**Comment by nritsche** [comment]
(CI tests fixed) @andrerenard I think this is ready. Some places imported `rte_branch_prediction.h` through `dpdkCore` and I decided to leave it that way.

File: lib/hsa/hsaPulsarUpdatePhase.cpp [hunk 1]
```diff
@@ -8,15 +8,12 @@
 #include "configUpdater.hpp"
 #include "hsaBase.h"
 #include "restServer.hpp"
+#include "visUtil.hpp"
```
Comment (cubranic) [inline lib/hsa/hsaPulsarUpdatePhase.cpp]: Ah, was the include "utils/visUtil.hpp" incorrect then? Or would both have worked?

File: lib/stages/frbPostProcess.cpp [hunk 2]
```diff
@@ -1,7 +1,3 @@
-// TODO Where do these live?
```
Comment (cubranic) [inline lib/stages/frbPostProcess.cpp]: I can't figure out which `#include` frbPostProcess gets the defines from?

Comment (andrerenard) [reply lib/stages/frbPostProcess.cpp]: `frbPostProcess` doesn't actually use `[un]likely` so it just doesn't need it.

File: lib/hsa/hsaPulsarUpdatePhase.cpp (hunk 1 above)
Comment (andrerenard) [reply lib/hsa/hsaPulsarUpdatePhase.cpp]: This also doesn't have any `[un]likely` uses it in, but I don't really know where the `visUtil.hpp` is coming from.  @nritsche was this included by mistake?

File: lib/stages/frbPostProcess.cpp (hunk 2 above)
Comment (cubranic) [reply lib/stages/frbPostProcess.cpp]: Hah, you're right! I remembered its use from pulsarPostProcess, and never noticed that it's not used in frbPostProcess even though the two are so similar. It probably should be, to mark the `if (startup)` case in line 48, but that's a different issue. :-)

File: lib/hsa/hsaPulsarUpdatePhase.cpp (hunk 1 above)
Comment (nritsche) [reply lib/hsa/hsaPulsarUpdatePhase.cpp]: I moved the visUtil import from the system includes to the project icludes. I guess both works, but style.

**Comment by nritsche** [comment]
@cubranic Both "testing/pulsarSimProcess.cpp" and "frbPostProcess.cpp" were not using the defines.

File: lib/hsa/hsaPulsarUpdatePhase.cpp (hunk 1 above)
Comment (andrerenard) [reply lib/hsa/hsaPulsarUpdatePhase.cpp]: Ah I see.

**Pull request merged**